Lanchonete/LanchoneteTests/Managers/AllIngridientsManagerTests.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  AllIngridientsManagerTests.swift
```

Comment:
```suggestion
//  AllIngredientsManagerTests.swift
```
And the file name too 😛

Reply:
Changed this